src/vm/docs/mod.rs

```diff
@@ -558,11 +599,15 @@ For strings and buffers, this function will return 1-length strings or buffers."
 };
 
 const INDEX_OF_API: SpecialAPI = SpecialAPI {
-    input_type: "buff|list A, buff|A",
+    input_type: "buff, (buff 1)
+list A, A
```

Comment:
Same as above - do we want the or syntax?

Reply:
see discussion above